domains/nfc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Tomgxz",
+        "email": "tomgxz138@gmail.com"
+    },
+    "record": {
+        "CNAME": "is3193tx.up.railway.app"
```

Comment:
Nothing is being displayed on site.

Reply:
This is due to how railway handles CNAME requests. The current development server is [](https://nfc-card.up.railway.app/tom_gxz)